src/doc/unstable-book/src/language-features/lang-items.md

```diff
@@ -11,8 +11,8 @@ it exists. The marker is the attribute `#[lang = "..."]` and there are
 various different values of `...`, i.e. various different 'lang
 items'.
 
-For example, `Box` pointers require two lang items, one for allocation
-and one for deallocation. A freestanding program that uses the `Box`
+For example, `Box` pointers require a lang item for allocation.
```

Comment:
I don't think this statement is accurate, at least currently, as `Box::new` is not implemented via a lang item, but via a builtin rustc attribute. The lang item the text probably refers to is `owned_box`, which is on the *type* `Box`. Note that in the future the statement might become true, as I am wondering about turning `Box::new` into a lang item so that rustc issues an error `#[rustc_box]` is put onto a function that is not `Box::new`.

Reply:
I'm pretty sure its referring to `exchange_malloc`, the lang item that `box`/`#[rustc_box]` call to allocate the memory.